packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -2101,4 +2101,40 @@ moduleFor('Components test: curly components', class extends RenderingTest {
 
     this.assertText('initial value');
   }
+
+  ['@htmlbars a two way binding flows upstream when set during init (#11192)']() {
```

Comment:
maybe something like: `ensure attributes which have been passed in can be mutated, when they are mutated the external source is notified`